pipelines/broad/dna_seq/germline/joint_genotyping/JointGenotyping.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 2.0.0
+2020-02-04
+
+Using Hail to instead of GenomicsDB 
```

Comment:
Seems like there's a word missing between "to" and "instead" - it would be great to have a touch more detail summarizing this change (what it is and why we're doing it)